Cannot log in with current cookies after long session.

I have setup the powerdns in runtime, after a while if I don't delete the cookies i get the following errors:

```
[2023-03-12 14:52:54,305] [_internal.py:224] INFO - 10.0.0.1 - - [12/Mar/2023 14:52:54] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.9/site-packages/werkzeug/middleware/proxy_fix.py", line 187, in __call__
    return self.app(environ, start_response)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2072, in wsgi_app
    ctx.push()
  File "/usr/local/lib/python3.9/site-packages/flask/ctx.py", line 434, in push
    self.session = session_interface.open_session(self.app, self.request)
  File "/usr/local/lib/python3.9/site-packages/flask_session/sessions.py", line 535, in open_session
    if saved_session and saved_session.expiry <= datetime.utcnow():
TypeError: '<=' not supported between instances of 'NoneType' and 'datetime.datetime'
[2023-03-12 14:52:54,576] [_internal.py:224] INFO - 10.0.0.1 - - [12/Mar/2023 14:52:54] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 304 -
[2023-03-12 14:52:54,579] [_internal.py:224] INFO - 10.0.0.1 - - [12/Mar/2023 14:52:54] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 304 -
[2023-03-12 14:52:55,591] [_internal.py:224] INFO - 10.0.0.1 - - [12/Mar/2023 14:52:55] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 304 -
[2023-03-12 14:52:55,664] [_internal.py:224] INFO - 10.0.0.1 - - [12/Mar/2023 14:52:55] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 304 -
```

This makes me unable to log in or access the DNS, the workaround i have found is either clear all cookies or accessing in incognito.

Same here. I had long session lifetime settings, so didn't notice it earlier. By setting short lifetime same errors occur and instance if giving http/500.
I'll try to debug it too... Looks like a non backend generic flask session code being used.
```
[2023-03-13 11:46:45,871] [app.py:1449] ERROR - Exception on /login [GET]
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/flask/app.py", line 2072, in wsgi_app
    ctx.push()
  File "/usr/lib/python3.10/site-packages/flask/ctx.py", line 434, in push
    self.session = session_interface.open_session(self.app, self.request)
  File "/usr/lib/python3.10/site-packages/flask_session/sessions.py", line 535, in open_session
    if saved_session and saved_session.expiry <= datetime.utcnow():
TypeError: '<=' not supported between instances of 'NoneType' and 'datetime.datetime'
[2023-03-13 11:46:45 +0000] [9] [ERROR] Error handling request /login
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/flask/app.py", line 2072, in wsgi_app
    ctx.push()
  File "/usr/lib/python3.10/site-packages/flask/ctx.py", line 434, in push
    self.session = session_interface.open_session(self.app, self.request)
  File "/usr/lib/python3.10/site-packages/flask_session/sessions.py", line 535, in open_session
    if saved_session and saved_session.expiry <= datetime.utcnow():
TypeError: '<=' not supported between instances of 'NoneType' and 'datetime.datetime'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/gunicorn/workers/sync.py", line 136, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/lib/python3.10/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/usr/lib/python3.10/site-packages/flask/app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3.10/site-packages/werkzeug/middleware/proxy_fix.py", line 187, in __call__
    return self.app(environ, start_response)
  File "/usr/lib/python3.10/site-packages/flask/app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3.10/site-packages/flask/app.py", line 1432, in handle_exception
    server_error = self.ensure_sync(handler)(server_error)
  File "/app/powerdnsadmin/routes/base.py", line 34, in handle_internal_server_error
    return render_template('errors/500.html', code=500, message=e), 500
  File "/usr/lib/python3.10/site-packages/flask/templating.py", line 153, in render_template
    ctx.app.update_template_context(context)
  File "/usr/lib/python3.10/site-packages/flask/app.py", line 750, in update_template_context
    context.update(func())
  File "/usr/lib/python3.10/site-packages/flask_login/utils.py", line 407, in _user_context_processor
    return dict(current_user=_get_user())
  File "/usr/lib/python3.10/site-packages/flask_login/utils.py", line 372, in _get_user
    current_app.login_manager._load_user()
  File "/usr/lib/python3.10/site-packages/flask_login/login_manager.py", line 362, in _load_user
    user_id = session.get("_user_id")
AttributeError: 'NoneType' object has no attribute 'get'
```

me to

Mar 13 16:13:09 webdns gunicorn[14646]: Traceback (most recent call last):
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/app.py", line 2072, in wsgi_app
Mar 13 16:13:09 webdns gunicorn[14646]:     ctx.push()
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/ctx.py", line 434, in push
Mar 13 16:13:09 webdns gunicorn[14646]:     self.session = session_interface.open_session(self.app, self.request)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask_session/sessions.py", line 535, in open_session
Mar 13 16:13:09 webdns gunicorn[14646]:     if saved_session and saved_session.expiry <= datetime.utcnow():
Mar 13 16:13:09 webdns gunicorn[14646]: TypeError: '<=' not supported between instances of 'NoneType' and 'datetime.datetime'
Mar 13 16:13:09 webdns gunicorn[14646]: [2023-03-13 16:13:09 +0300] [14646] [ERROR] Error handling request /
Mar 13 16:13:09 webdns gunicorn[14646]: Traceback (most recent call last):
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/app.py", line 2072, in wsgi_app
Mar 13 16:13:09 webdns gunicorn[14646]:     ctx.push()
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/ctx.py", line 434, in push
Mar 13 16:13:09 webdns gunicorn[14646]:     self.session = session_interface.open_session(self.app, self.request)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask_session/sessions.py", line 535, in open_session
Mar 13 16:13:09 webdns gunicorn[14646]:     if saved_session and saved_session.expiry <= datetime.utcnow():
Mar 13 16:13:09 webdns gunicorn[14646]: TypeError: '<=' not supported between instances of 'NoneType' and 'datetime.datetime'
Mar 13 16:13:09 webdns gunicorn[14646]: During handling of the above exception, another exception occurred:
Mar 13 16:13:09 webdns gunicorn[14646]: Traceback (most recent call last):
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/gunicorn/workers/sync.py", line 136, in handle
Mar 13 16:13:09 webdns gunicorn[14646]:     self.handle_request(listener, req, client, addr)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
Mar 13 16:13:09 webdns gunicorn[14646]:     respiter = self.wsgi(environ, resp.start_response)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/app.py", line 2091, in __call__
Mar 13 16:13:09 webdns gunicorn[14646]:     return self.wsgi_app(environ, start_response)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/werkzeug/middleware/proxy_fix.py", line 187, in __call__
Mar 13 16:13:09 webdns gunicorn[14646]:     return self.app(environ, start_response)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/app.py", line 2076, in wsgi_app
Mar 13 16:13:09 webdns gunicorn[14646]:     response = self.handle_exception(e)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/app.py", line 1432, in handle_exception
Mar 13 16:13:09 webdns gunicorn[14646]:     server_error = self.ensure_sync(handler)(server_error)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/powerdnsadmin/routes/base.py", line 34, in handle_internal_server_error
Mar 13 16:13:09 webdns gunicorn[14646]:     return render_template('errors/500.html', code=500, message=e), 500
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/templating.py", line 153, in render_template
Mar 13 16:13:09 webdns gunicorn[14646]:     ctx.app.update_template_context(context)
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask/app.py", line 750, in update_template_context
Mar 13 16:13:09 webdns gunicorn[14646]:     context.update(func())
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask_login/utils.py", line 407, in _user_context_processor
Mar 13 16:13:09 webdns gunicorn[14646]:     return dict(current_user=_get_user())
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask_login/utils.py", line 372, in _get_user
Mar 13 16:13:09 webdns gunicorn[14646]:     current_app.login_manager._load_user()
Mar 13 16:13:09 webdns gunicorn[14646]:   File "/opt/web/powerdns-admin/venv/lib/python3.9/site-packages/flask_login/login_manager.py", line 362, in _load_user
Mar 13 16:13:09 webdns gunicorn[14646]:     user_id = session.get("_user_id")
Mar 13 16:13:09 webdns gunicorn[14646]: AttributeError: 'NoneType' object has no attribute 'get'
Mar 13 16:13:10 webdns gunicorn[14646]: [2023-03-13 16:13:10,011] [app.py:1449] ERROR - Exception on / [GET]

--- a/powerdnsadmin/routes/index.py
+++ b/powerdnsadmin/routes/index.py
@@ -528,7 +528,6 @@ def clear_session():
     session.pop('google_token', None)
     session.pop('authentication_type', None)
     session.pop('remote_user', None)
-    session.clear()
     logout_user()
 
 

